### DISCUSSION TITLE
Question about step forms

### DISCUSSION THREAD

**Discussion by edrisi6** [Budibase/budibase#8273] [Help]
I want to build a step form that feeds different tables. I am not sure if my approach is correct. Here is what I have done:

1. Created all tables I need (account, orders..) 
2. I created a screen for each table then used a form component linked to it then used a step or two under each form
3. I plan to use the navigate function through the "Next" Button actions to jump to the next step using the step number

would this work?

**Comment by melohagan** [comment]
Hey @edrisi6 

This sounds reasonable, although you could have a single screen with one form and many steps that saves to a different table in each step, or to all the different tables at the very end. 
The choice is in your hands. There guides may help:

 - https://docs.budibase.com/docs/form-steps
 - https://docs.budibase.com/docs/referencing#example-creating-and-linking-two-table-entries-in-one-form
